testing/functional/tests/scheduler/dragAndDrop/init/widget.data.ts

```diff
@@ -0,0 +1,24 @@
+
+export const dataSource = [
+    {
+        Id: 1,
+        text: 'Appointment #1',
```

Comment:
Rename appointments in tests, so it will be easier to understand which of them is used in test